test/data_tests/cassiopeia_tree_test.py

```diff
@@ -1664,6 +1664,35 @@ def test_set_dissimilarity(self):
                 "node3", {leaf: 0 for leaf in tree_no_map.leaves}
             )
 
+    def test_get_depth(self):
+        tree = cas.data.CassiopeiaTree(tree=self.test_network)
+        self.assertEqual(tree.get_depth(), 8)
+
```

Comment:
I'm pretty sure this operates exactly as `get_max_depth_of_tree`

Reply:
Addressed in a previous comment.